config/publishers/bitbucket.md

```diff
@@ -23,16 +23,18 @@ module.exports = {
       name: '@electron-forge/publisher-bitbucket',
       config: {
         repository: {
-          owner: 'myusername',
-          name: 'myreponame'
+            owner: 'myusername',
```

Comment:
dont think this whitespace is necessary

we were formatting w/ 2 space tabs

Reply:
You're right. Looks like eslint decided to auto-format it.